src/modules/sensors/sensor_params_mag.c

```diff
@@ -53,6 +53,15 @@
  * @value 63 Six side calibration
  * @group Sensors
  */
+PARAM_DEFINE_INT32(SENS_MAG_SIDES, 63);
+
+/**
+ * For legacy QGC support only
+ *
+ * Use SENS_MAG_SIDES instead
+ *
+ * @group Sensors
```

Comment:
We could hide it in a different category?

```suggestion
 * @group Sensors
 * @category Developer
```

Reply:
Good idea. https://github.com/PX4/PX4-Autopilot/pull/20919